utils/pod/pod.go

```diff
@@ -763,6 +802,21 @@ func SendMesosStatus(driver executor.ExecutorDriver, taskId *mesos.TaskID, state
 		TaskId: taskId,
 		State:  state,
 	}
+
+	logStatus := waitUtil.GetLogStatus()
+	log.Debugf("Log status is : %v", logStatus)
+	log.Debugf("Task status is : %v", state.Enum().String())
+
+	if logStatus == false {
+		if state.Enum().String() == mesos.TaskState_TASK_FINISHED.Enum().String() ||
```

Comment:
Curious, can you use a switch statement here instead? Also, do we need to use the string form to compare? Can we compare the enums directly?